kernel/pipedev.c

```diff
@@ -580,7 +634,7 @@ static int _pd_fcntl(
     {
         case F_SETFL:
         {
-            pipe->shared->flags |= arg;
+            pipe->flags |= arg;
```

Comment:
Should F_GETFL be serviced by the kernel, instead of invoke the host? How about other fcntl cmds? If a pipe fcntl cmd is invalid or not supported by Mystikos implementation, it's better to error out inside the kernel, instead of exposing the info to the host.

Reply:
Fixed. F_GETFL is no completely serviced by the kernel. And now we error out if these flags are not supported.